winthrop/books/migrations/0012_add_book_slug.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-02 19:55
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.utils.text import slugify
+from unidecode import unidecode
+
+
+def generate_book_slug(book):
```

Comment:
codecov is angry that this method isn't tested, which is a bummer because the model method version of it *is* tested.